console/src/app/services/new-feature.service.ts

```diff
@@ -11,4 +16,13 @@ export class NewFeatureService {
   public getInstanceFeatures(): Promise<GetInstanceFeaturesResponse> {
     return this.grpcService.featureNew.getInstanceFeatures({});
   }
+
+  public setInstanceFeatures(req: SetInstanceFeaturesRequest): Promise<SetInstanceFeaturesResponse> {
```

Comment:
Ah i see why the create is needed this type can be MessageInitType<typeof SetInstanceFeaturesResponseSchema>. You can check the UserService to see how it is done. At the moment we still call create inside the Service but that is also not needed.

Reply:
done